CONTRIBUTING.md

```diff
@@ -38,52 +38,41 @@ Finally, set the upstream remote to the official Spyder repo with:
 $ git remote add upstream https://github.com/spyder-ide/spyder.git
 ```
 
-### Creating a conda environment or virtualenv
 
-If you use Anaconda you can create a conda environment with the following commands:
+### Creating an environment and installing dependencies
 
-```bash
-$ conda create -n spyder-dev python=3
-$ conda activate spyder-dev
-```
-
-You can also use `virtualenv` on Linux, but `conda` is **strongly** recommended:
+If you use a Conda-based distribution (Anaconda, etc), you can create an environment and install the necessary dependencies as follows:
 
 ```bash
-$ mkvirtualenv spyder-dev
-$ workon spyder-dev
+$ conda create -n spyder-dev -c conda-forge --file requirements/conda.txt
+$ conda activate spyder-dev
+(spyder-dev) $ python install_dev_repos.py
 ```
 
-
-### Installing dependencies
-
-After you have created your development environment, you need to install Spyder's necessary dependencies. The easiest way to do so (with Anaconda) is
+If you are running on macOS, you will also need to install `python.app`, so after activating the environment, run
 
 ```bash
-$ conda install -c conda-forge --file requirements/conda.txt
+$ conda install -c conda-forge python.app
 ```
 
-This installs all Spyder's dependencies into the environment.
-If you are running on macOS, you will also need to install `python.app`.
+You can also use `virtualenv` on Linux, but `conda` is **strongly** recommended:
 
 ```bash
-$ conda install python.app
+$ mkvirtualenv spyder-dev
+$ workon spyder-dev
+(spyder-dev) $ pip install -e .
+(spyder-dev) $ python install_dev_repos.py --no-install spyder
```

Comment:
Are these two lines really necessary? Why do you need Spyder in development mode and not the subrepos when working with virtualenvs?

Reply:
This line

```suggestion
(spyder-dev) $ pip install -e .
```

should be retained install Spyder's other deps first, analogous to installing `--file requirements/conda.txt` in the conda env, and then the bootstrap script installs the subrepos in editable mode, at least following the conda example above.

Reply:
`install_dev_repos` will install the the subrepos in develop mode; but as @CAM-Gerlach mentioned, `bootstrap.py` will now do this by default if they are not already installed. There is no need for the user to perform the extra step to install them.